frame/staking/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-staking"
-version = "2.0.0"
+version = "0.8.0"
```

Comment:
staking has been downgraded, 2.0 should include https://github.com/paritytech/substrate/issues/4407